backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlQuery.java

```diff
@@ -1,7 +1,6 @@
 package org.molgenis.emx2.sql;
```

Comment:
restructured this file so more changes than needed but I hope will make this huge file more easy to understand